plugins/PagePerformance/javascripts/rowaction.js

```diff
@@ -33,20 +33,21 @@
 
     DataTable_RowActions_PagePerformance.prototype.performAction = function (label, tr, e, originalRow) {
         var apiMethod = this.dataTable.param.module + '.' + this.dataTable.param.action;
-        this.openPopover(apiMethod, label);
+        this.openPopover(apiMethod, label, this.dataTable.param.flat || 0);
```

Comment:
not a functional change but looks a bit more consistent with the rest of the change
```suggestion
        var isReportFlat = this.dataTable.param.flat || 0;
        this.openPopover(apiMethod, label, isReportFlat);
```